ocaml/xapi/xapi_network.mli

```diff
@@ -70,7 +70,15 @@ val pool_introduce :
   name_label:string ->
   name_description:string ->
   mTU:int64 ->
-  other_config:(string * string) list -> bridge:string -> [ `network ] Ref.t
+  other_config:(string * string) list ->
+  bridge:string ->
+  managed:bool -> [ `network ] Ref.t
+
+(** Choose a bridge name when create network *)
+val choose_bridge_name :
```

Comment:
Don't add this function to the module interface here. It is local to the Xapi_network module and not to be used outside.